README.md

```diff
@@ -203,6 +211,26 @@ START_AT_STEP=git-bake ./enki ./data/tin-bk all-pdf
 **Note:** The arguments following all-pdf can be omitted since they are only used in the initial step
 
 
+## Debugging
+
+### Python Debugging
+
+In order to debug a python script from the `./bakery-src/scripts` folder, you can add the `--pdb` flag to the `./enki` command. This will open the `5678` port on the container and allow you to connect to it with a debugger.
+In addition, you should add the following script on top of the python script you want to debug and set breakpoints below the `debug()` line:
+
+```python
+from .profiler import debug
+debug()
+```
+
+### Java Debugging
+
+In order to debug the Java code, you can set the breakpoints in the Java code and then run the `./enki` command with the `--jdb` flag. The java program will then suspend till the client is connected to it and stop at the breakpoints and you can step through the code.
+
+### Javascript Debugging
+
+In order to debug the Javascript code, you can set the breakpoints in the Javascript code under `bakery-js/src` and then run the `./enki` command with the `--jsdb` flag. The node program will then suspend till the client is connected to it and stop at the breakpoints and you can step through the code.
```

Comment:
I was unable to set breakpoints in files within `bakery-js/src/`.

I was however able to set breakpoints in `bakery/dist/` after running `npm install && npm run build` inside bakery-js. Maybe there is a way (via commandline) to tell the node process to use sourcemaps when given breakpoints?